### PR TITLE
add section projects

### LINKED ISSUE #5: projects-section
Section of projects includes:
- title
- short sentence describes the projects
- four buttons for four different projects categories
- gallery of projects pictures

### PR DESCRIPTION
**Pull request by ahmedhamouda2**
Realtes #5

### CODE REVIEW

**Review by Ibrahim-Jarada — DISMISSED** [review]

**Review by Ola014 — APPROVED** [review]

**Review by Ola014 — APPROVED** [review]

File: index.css [hunk 1]
```diff
@@ -177,12 +177,8 @@ hr {
...
     text-align: center;
-    padding-right: 15px;
-    padding-left: 15px;
-    margin-right: auto;
-    margin-left: auto;
-    margin-top: 50px;
-
+    padding: 0 15px;
+    margin: 50px auto;
 }
```
Comment (Ola014) [inline index.css:182]: There is two code lines repeated or overrided

File: index.html [hunk 2]
```diff
@@ -71,5 +71,24 @@ <h2>commercial design</h2>
       <div class="clearix"></div>
     </section>
   <!-- End secrion our services -->
+
+    <!-- start section projects -->
+    <section class="projects">
+        <h1>Projects</h1>
+        <p class="outside-para">Lorem ipsum dolor sit amet consectetur.</p>
+        <img src="img/project/01.jpg" alt="">
```
Comment (sara219) [inline index.html]: add the alt value

File: index.css [hunk 3]
```diff
@@ -264,4 +260,68 @@ hr {
...
+    padding-top: 30px;
+}
+.projects h1::after {
+    content: " ";
+    display: block;
+    position: absolute;
+    top: 70px;
+    left: 49%;
+    width: 50px;
+    border-bottom: 3px solid #a7c44c;
```
Comment (sara219) [inline index.css:287]: Screenshot from 2021-03-02 20-27-05

See how the border appears below it, not in the middle

**Review by sara219 — CHANGES_REQUESTED** [review]

**Review by Ola014 — CHANGES_REQUESTED** [review]

File: index.html [hunk 4]
```diff
@@ -71,5 +71,24 @@ <h2>commercial design</h2>
       <div class="clearix"></div>
     </section>
   <!-- End secrion our services -->
+
+    <!-- start section projects -->
+    <section class="projects">
+        <h1>Projects</h1>
+        <p class="outside-para">Lorem ipsum dolor sit amet consectetur.</p>
```
Comment (sara219) [inline index.html]: don't use dummy text, it's better to use real text

**Review by Ola014 — DISMISSED** [review]

File: index.css [hunk 5]
```diff
@@ -264,4 +260,68 @@ hr {
...
+@media (max-width:767px) {
+    .projects img {
+        width: 100%;
+        margin: 20px auto;
+    }
+    .projects h1::after {
+        left: 47%;
+    }
+    .projects img {
+        width: 100%;
```
Comment (sara219) [inline index.css]: pay attention to the code formating, every line start from diff point 
use prettier correctly

**Review by sara219 — CHANGES_REQUESTED** [review]

File: index.css [hunk 6]
```diff
@@ -264,4 +260,68 @@ hr {
...
+    left: 49%;
+    width: 50px;
+    border-bottom: 3px solid #a7c44c;
+}
+.projects img {
+    width: 25%;
+    height: 250px;
+    border-radius: 0 70px;
+    margin-left: 3%;
+    margin-top: 20px;
```
Comment (sara219) [inline index.css]: you use % for the left margin, and you use px for the other 
choose one of them

**Review by sara219 — CHANGES_REQUESTED** [review]

**Review by Ibrahim-Jarada — APPROVED** [review]

**Review by sara219 — APPROVED** [review]

**Pull request merged**